### ISSUE TITLE
Display file and total project size limits on project settings page

### ISSUE THREAD

**Issue by di** [pypa/warehouse#8187]
We should display the current file size limit (`Project.upload_limit`) and total size limit (`Project.total_size_limit`)
on the project settings page for all projects, with external links to https://pypi.org/help/#file-size-limit and https://pypi.org/help/#project-size-limit, respectively.

---

**Good First Issue**: This issue is good for first time contributors. If you've already contributed to Warehouse, work on [another issue without this label](https://github.com/pypa/warehouse/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22good+first+issue%22) instead. If there is not a corresponding pull request for this issue, it is up for grabs. For directions for getting set up, see our [Getting Started Guide](https://warehouse.pypa.io/development/getting-started/).

If you are working on this issue and have questions, feel free to ask them here, in the [`#pypa-dev` chat channel on Freenode](https://webchat.freenode.net/?channels=%23pypa-dev), or on the [distutils-sig.python.org mailing list](https://mail.python.org/mailman3/lists/distutils-sig.python.org/).

**Screenshot Required**: *If your pull request makes a visual change*, include a screenshot of your update. This helps our team give you feedback faster.

**Comment by imsahil007** [comment]
Hey, @di I have updated the template for this. Which file do I have to update for updating the text value inside the anchor tag?

**Comment by di** [comment]
Re-opening this, I missed that in #8221 it is showing `0 bytes` instead of the default values when a limit is not set.

@imsahil007, can you take a look?

**Comment by imsahil007** [comment]
> Re-opening this, I missed that in #8221 it is showing `0 bytes` instead of the default values when a limit is not set.
> 
> @imsahil007, can you take a look?

I missed that! I checked the values only after updating the limits. I will look into that.

**Comment by imsahil007** [comment]
![Screenshot from 2020-07-08 12-14-13](https://user-images.githubusercontent.com/35963992/86886484-a29e2000-c114-11ea-9b02-04c9f593d476.png)
I think this will close the issue for now.
The default values in the database are set to 0. Even in the admin panel if the value is not there we are printing the warehouse.forklift constants MAX_FILESIZE and so... I used the same approach.
